android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -921,7 +924,11 @@ public void setAttributedStringHint(AttributedStringProxy attrString)
 	{
 		Spannable spannableText = AttributedStringProxy.toSpannable(attrString, TiApplication.getAppCurrentActivity());
 		if (spannableText != null) {
-			int type = getProxy().getProperties().getInt(TiC.PROPERTY_HINT_TYPE);
+			int type = UIModule.HINT_TYPE_STATIC;
+			KrollProxy proxy = getProxy();
```

Comment:
another potential place we could return a null object variant? i.e. if a TiUIView's proxy is null, return a special NullProxy instance that has empty properties?